code/modules/food&drinks/food/snacks_meat.dm

```diff
@@ -124,6 +124,21 @@
 	desc = "Severed lizard tail on a stick."
 	bonus_reagents = list("nutriment" = 1, "vitamin" = 4)
 
+/obj/item/weapon/reagent_containers/food/snacks/rawkhinkali
+	name = "raw khinkali"
+	desc = "One hundred khinkalis? Do I look like a pig?"
+	icon_state = "khinkali"
+	list_reagents = list("nutriment" = 1, "vitamin" = 1)
+	cooked_type = /obj/item/weapon/reagent_containers/food/snacks/khinkali
+
+/obj/item/weapon/reagent_containers/food/snacks/khinkali
+    name = "khinkali"
```

Comment:
Please don't indent your lines using the spacebar, the TAB key is the codebase standard for indenting lines.